### PR TITLE
Fix class docblocks

### PR DESCRIPTION
**Pull request by lucasmichot**
Fix class docblocks to respect `phpdoc_scalar`

### CODE REVIEW

**Comment by lucasmichot** [comment]
cc @GrahamCampbell nothing detected with `PHP CS Fixer version 1.10.2 by Fabien Potencier (e8b3c4e)`
But issues in #497

**Comment by GrahamCampbell** [comment]
:-1:

**Comment by GrahamCampbell** [comment]
This is actually a bug I accidentally introduced in the deploy last Sunday.

**Comment by GrahamCampbell** [comment]
> nothing detected with PHP CS Fixer version 1.10.2 by Fabien Potencier 

Yeh, we don't just run php-cs-fixer as is.

File: src/Carbon/Carbon.php [hunk 1]
```diff
@@ -31,23 +31,24 @@
...
- * @property-read integer $offset the timezone offset in seconds from UTC
- * @property-read integer $offsetHours the timezone offset in hours from UTC
- * @property-read boolean $dst daylight savings time indicator, true if DST, false otherwise
- * @property-read boolean $local checks if the timezone is local, true if local, false otherwise
- * @property-read boolean $utc checks if the timezone is UTC, true if UTC, false otherwise
- * @property-read string  $timezoneName
- * @property-read string  $tzName
+ * @property      \DateTimeZone $timezone the current timezone
+ * @property      \DateTimeZone $tz alias of timezone
+ *
```
Comment (GrahamCampbell) [inline src/Carbon/Carbon.php]: the addition of this newline is a bug in styleci

**Comment by GrahamCampbell** [comment]
Oh, wait, did you add the newline yourself?

**Comment by GrahamCampbell** [comment]
Meh, maybe it's not a bug. What do you think?

**Comment by GrahamCampbell** [comment]
In terms of fixing integer in property-read, that is a new features added in the last deploy.

**Comment by lucasmichot** [comment]
@GrahamCampbell see https://styleci.io/analyses/qB7O7z

**Comment by GrahamCampbell** [comment]
Maybe it's fine.

**Comment by GrahamCampbell** [comment]
My :-1: more like a :+1: now I look at the details agian, lol.

**Comment by GrahamCampbell** [comment]
I just need to know if you think all `property`ish tags should be collected or not?

**Comment by GrahamCampbell** [comment]
https://i.starbs.net/XRY6

**Comment by GrahamCampbell** [comment]
It basically means I need to add a new group in here.

**Comment by GrahamCampbell** [comment]
Like this: https://i.starbs.net/5zYw.

**Comment by lucasmichot** [comment]
> I just need to know if you think all propertyish tags should be collected or not?

IMO they should yes

**Comment by GrahamCampbell** [comment]
I'll update StyleCI again next Sunday then.

**Comment by lucasmichot** [comment]
cool :+1:

**Pull request merged**